src/app/docs/toolbar/toolbar-guidelines.component.html

```diff
@@ -0,0 +1,285 @@
+        <sky-definition-list-content *ngFor="let item of shareButtonInfo">
+          <sky-definition-list-label>
+            {{item.label}}
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{item.value}}
+          </sky-definition-list-value>
+        </sky-definition-list-content>
+      </sky-definition-list>
+    </sky-column>
+  </sky-row>
+
+  <div class="sky-emphasized">Export button</div>
+  <sky-row>
+    <sky-column screenSmall="3" screenMedium="2">
+      <div style="margin: 5px 0;"><sky-dropdown><sky-dropdown-button><i class="fa fa-file-o" style="margin-right: 5px;"></i>Export</sky-dropdown-button></sky-dropdown></div>
+      <div style="margin: 5px 0;"><button class="sky-btn sky-btn-default"><i class="fa fa-file-excel-o" style="margin-right: 5px;"></i>Export</button></div>
+      <div style="margin: 5px 0;"><button class="sky-btn sky-btn-default"><i class="fa fa-file-pdf-o" style="margin-right: 5px;"></i>Export</button></div>
+    </sky-column>
+    <sky-column screenSmall="9" screenMedium="10">
+      <sky-definition-list>
+        <sky-definition-list-content *ngFor="let item of exportButtonInfo">
+          <sky-definition-list-label>
+            {{item.label}}
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{item.value}}
+          </sky-definition-list-value>
+        </sky-definition-list-content>
+      </sky-definition-list>
+    </sky-column>
+  </sky-row>
+
+<sky-docs-design-guideline
+  heading="More actions"
+>
+  <p>
+    Place less important or less-frequently used actions in a more actions menu. And when horizontal space in a toolbar is insufficient for all actions, standard actions should collapse into the more actions menu.
+  </p>
+
+  <div class="sky-emphasized">More button</div>
+  <sky-row>
+    <sky-column screenSmall="3" screenMedium="2">
+      <div style="margin: 5px 0;">
+				<sky-dropdown>
+          <sky-dropdown-button>
+						<i class="fa fa-ellipsis-h" style="margin-right: 5px;"></i> More
+          </sky-dropdown-button>
+					</sky-dropdown>
+			</div>
+    </sky-column>
+    <sky-column screenSmall="9" screenMedium="10">
+      <sky-definition-list>
+        <sky-definition-list-content *ngFor="let item of moreButtonInfo">
+          <sky-definition-list-label>
+            {{item.label}}
+          </sky-definition-list-label>
+          <sky-definition-list-value>
+            {{item.value}}
+          </sky-definition-list-value>
+        </sky-definition-list-content>
+      </sky-definition-list>
+    </sky-column>
+  </sky-row>
+</sky-docs-design-guideline>
+
+<sky-docs-design-guideline
+  heading="Find in this list"
+>
+  <p>
+    When a toolbar includes a search component, it should follow the actions and more actions button. The search field is aligned with the action buttons, so if a user has a really large monitor resolution, the field appears on the left side of the screen.
```

Comment:
I'd change the first sentence to `Place the search component to the right of the standard toolbar actions and the more actions button.` 

I'd tighten up the second sentence to `Align the search field with the action buttons so that in large monitor resolutions, it appears on the left side of the screen.` What do you think?

Reply:
Just following up on these suggestions. Do you think they are worth making?